### PR TITLE
Added new command for Pogy

### PR DESCRIPTION
**Pull request by Boeing-737-800**
This command can moderate a user nickname in one millisecond.

### CODE REVIEW

File: src/commands/moderation/modnick.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    const client = message.client
+    const logging = await Logging.findOne({ guildId: message.guild.id });
+    
+    const member =
+      message.mentions.members.last() ||
+      message.guild.members.cache.get(args[0]);
+
+    const reason = args.slice(1).join(" ");
+
+    const impostorpassword = darkpassword.generate({
```
Comment (fb-sean) [inline src/commands/moderation/modnick.js:32]: Don't use an package for an simple random ID String. For Example use this:

`Math.random().toString(36).substring(2, 5) + Math.random().toString(36).substring(2, 5)`

**Review by fb-sean — COMMENTED** [review]
This really need some improvements and also make sure to check if the user is even editable/manageable before changing a nick.

**Comment by Boeing-737-800** [comment]
Command has been fixed.

**Comment by fb-sean** [comment]
Still no right permission check please add an if for this property and check if the user is manageable.

=> https://discord.js.org/#/docs/discord.js/13.6.0/class/GuildMember?scrollTo=manageable

**Comment by Boeing-737-800** [comment]
Command already checks if a user is manageable (via role positions and if the user pinged themselves)

**Pull request closed without merging**